mantle/cmd/plume/prerelease.go

```diff
@@ -94,16 +94,18 @@ func AddSpecFlags(flags *pflag.FlagSet) {
 	board := sdk.DefaultBoard()
 	flags.StringVarP(&specBoard, "board", "B", board, "target board")
 	flags.StringVarP(&specChannel, "channel", "C", "testing", "target channel")
-	flags.MarkDeprecated("channel", "use --stream instead")
+	if err := flags.MarkDeprecated("channel", "use --stream instead"); err != nil {
+		plog.Error(err)
```

Comment:
Could probably make this a `panic`. Looking at https://github.com/spf13/pflag/blob/85dd5c8bc61cfa382fecd072378089d4e856579d/flag.go#L411, this can only happen due to programmer error.

Reply:
Thank you for review. Fixed.